chromadb/test/property/test_embeddings.py

```diff
@@ -868,3 +868,20 @@ def test_0dim_embedding_validation() -> None:
         "Expected each embedding in the embeddings to be a 1-dimensional numpy array with at least 1 int/float value. Got a 1-dimensional numpy array with no values at pos"
         in str(e)
     )
+
+def test_no_op_compaction(client: ClientAPI) -> None:
+    reset(client)
+    coll = client.create_collection(name="noop")
+    coll.delete(ids=[str(i) for i in range(5000)])
+    wait_for_version_increase(client, coll.name, get_collection_version(client, coll.name), 240)
+
+def test_add_then_purge(client: ClientAPI) -> None:
```

Comment:
Can we extend this test to try to query after the second wait?

Reply:
Added an assertion that no records should remain after the second wait